configs/mvp/campus_config/mvp_campus.py

```diff
@@ -171,16 +171,51 @@
         ),
     ),
     evaluation_setup=dict(
-        type='MVPEvaluation',
+        type='End2EndEvaluation',
         dataset_name=__test_dataset__,
         pred_kps3d_convention='campus',
         gt_kps3d_convention='campus',
+        eval_kps3d_convention='campus',
         n_max_person=__n_instance__,
-        selected_limbs_name=[
-            'left_lower_leg', 'right_lower_leg', 'left_upperarm',
-            'right_upperarm', 'left_forearm', 'right_forearm', 'left_thigh',
-            'right_thigh'
+        checkpoint_select='pcp_total_mean',
```

Comment:
What does the `checkpoint_select` mean, is it the metric the user wants to calculate? In my opinion, isn't `pick_dict` the metric chosen by the user to calculate?

Reply:
For the learning-based method, evaluation is called for testing in every epoch during training.`checkpoint_select` is to return one metric from `pick_dict` that helps to select the best checkpoint during training. `pick_dict` is the metric chosen by the user to calculate and print.